source/extensions/common/wasm/wasm.h

```diff
@@ -54,10 +56,31 @@ using WasmCallback3Void = void (*)(void*, uint32_t, uint32_t, uint32_t);
 using WasmCallback4Void = void (*)(void*, uint32_t, uint32_t, uint32_t, uint32_t);
 using WasmCallback5Void = void (*)(void*, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t);
 using WasmCallback0Int = uint32_t (*)(void*);
+using WasmCallback1Int = uint32_t (*)(void*, uint32_t);
+using WasmCallback2Int = uint32_t (*)(void*, uint32_t, uint32_t);
 using WasmCallback3Int = uint32_t (*)(void*, uint32_t, uint32_t, uint32_t);
 using WasmCallback5Int = uint32_t (*)(void*, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t);
 using WasmCallback9Int = uint32_t (*)(void*, uint32_t, uint32_t, uint32_t, uint32_t, uint32_t,
                                       uint32_t, uint32_t, uint32_t, uint32_t);
+// Using the standard g++/clang mangling algorithm:
+using WasmCallback_Zjl = void (*)(void*, uint32_t, int64_t);
```

Comment:
Could you add the comment what Zjl Zjm mjj are referring to?

Reply:
Done.